publisher-sdk/proguard-rules.pro

```diff
@@ -99,10 +99,6 @@
 
 -keep public interface com.criteo.publisher.CriteoBannerAdListener {*;}
 
--keep public interface com.criteo.publisher.CriteoInterstitialAdListener {*;}
```

Comment:
Why are you removing `CriteoInterstitialAdListener`? I thought it was the new interface that publishers need to implement?

Reply:
I'm using `@Keep` instead. Personally, I find it better than proguard rules as:
- it is not tight to any particular techno
- it is really next to the code declaration, so public API are explicit when reading the code.

If you prefer proguard rules, I can revert back this.

Reply:
I am fine with `@Keep` but in this case, we should move everything to `Keep` that way it would be less confusing.

Reply:
Done: EE-1248